content/en/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -11,9 +11,8 @@ tags:
 - core-object
 - storage
 ---
- Claims storage resources defined in a PersistentVolume so that it can be mounted as a volume in a container.
+ Claims storage resources defined in a {{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}} so that it can be mounted as a volume in a {{< glossary_tooltip text="container" term_id="container" >}}.
```

Comment:
nit: line 14, Though this PR is about cross references and tooltips, this sentence is hard to read. 
From the concept page:
A PersistentVolumeClaim (PVC) is a request for storage by a user.

`Claims as volumes`:
Pods access storage by using the claim as a volume. Claims must exist in the same namespace as the Pod using the claim. The cluster finds the claim in the Pod’s namespace and uses it to get the PersistentVolume backing the claim. The volume is then mounted to the host and into the Pod.